azext_iot/device_certification/_help.py

```diff
@@ -0,0 +1,173 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Help definitions for Device Certification commands.
+"""
+
+from knack.help_files import helps
+
+
+def load_help():
+    helps["iot device-certification"] = """
```

Comment:
Given the current root command space as-is....it would be beneficial to have it as `az iot device certification` rather than `az iot device-certification`. We have an existing root command group that `certification` could plug into and seems like the right theme/category. `az iot device` currently is all about device impersonation and interactions as a device. Giving a sub command group around certifying devices would be a good fit.

Reply:
Also will have a cleaner user experience when exploring commands since there won't be a "device-certification" injected at the root `az iot` 

![image](https://user-images.githubusercontent.com/28658112/87483469-a3d5be00-c5e8-11ea-8af1-a18d2597f6fb.png)

Reply:
Of course we'd update the current description for the command group.

Reply:
I agree that "device-certification" is likely the wrong name as this is more "product-certification". It has much less to do with the existing device commands and more to do with setting up the certification service (AICS) to perform testing of an individual device that is a representative of the product line. So a single Contoso ThermostatX gets tested for Contoso ThermostatX product line to get a certified badge in Device Catalog.

Reply:
What are your thoughts on `az iot product` as this namespace? Then all the commands become `az iot product requirement`, `az iot product test`, `az iot product test-case`, etc?  Also, do you think `test-case`, `test-run`, and `test-task` should become subgroups under test? `az iot product test task create` for example

Reply:
Note: we talked about this offline.